src/mpl/src/gavl/mpl_gavl.c

```diff
@@ -282,10 +345,10 @@ int MPL_gavl_tree_search(MPL_gavl_tree_t gavl_tree, const void *addr, uintptr_t
 {
```

Comment:
Ideally I want `MPL_gavl_tree_search` be statically inlined. There is no reason why it cannot. It might give me visible overhead on KNL (I see constant overhead 1us in latency test on Theta when using this routine with `MPI_Put` in my dynamic window patch, but I am still working on more analysis to ensure the cause).

Reply:
You don't have to deal with the above comment in this PR. But I want you know this concern.

Reply:
It is public API, how can we make it `static inline`?
You mean move it to header file?

Reply:
Yes.